web/src/app/sidebar/interested-courses/component.ts

```diff
@@ -41,7 +41,7 @@ export class InterestedCoursesComponent implements OnInit {
     // display interested courses on sidebar
     // display message to try selecting some if none
     if (this.listingIds.size > 0) {
-      this.statusText = '';
+      document.getElementById('sidebar-stat').classList.add("hide-sidebar-status");
```

Comment:
You shouldn't use the `document` selector functions without a very compelling reason. In general you should not be writing logic to edit the view in a controller. To conditionally display an element, in general use `*ngIf`.